### PR TITLE
can-i-please-get-a-puppy

### PR DESCRIPTION
**Pull request by leggsimon**
So the world can see it @adgad  🐿 v2.7.0

mar-23-2018 11-00-33

### CODE REVIEW

File: components/n-ui/roe/can-i-please-get-a-puppy.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const canIPleaseGetAPuppy = () => {
+	const puppyId = 'canIPleaseGetAPuppy';
+	const isPuppyArticle = document.documentElement.getAttribute('data-content-id') === 'b1a49898-2c44-11e8-a34a-7e7563b0b0f4';
```
Comment (adgad) [inline components/n-ui/roe/can-i-please-get-a-puppy.js]: could this just be on _every_ article? 🙈

Comment (leggsimon) [reply components/n-ui/roe/can-i-please-get-a-puppy.js]: haha! sure

**Comment by bondz** [comment]
https://twitter.com/bhgreeley/status/976459499920920576/

LGTM 👇 
https://twitter.com/bhgreeley/status/976963525477355521

**Comment by doramatadora** [comment]
👏 👏 👏 
❤️

**Comment by chee** [comment]
> She's finally on the schedule in May to read bedtime stories to the dogs

> last month she donated $62 of her own allowance.

**Comment by leggsimon** [comment]
😢 🐶 👋

**Comment by chee** [comment]
puppy no

**Comment by adgad** [comment]
I'd like to think we played a small part in his recruitment.

**Comment by chee** [comment]
puppie's face

**Pull request closed without merging**